src/test/java/com/r307/arbitrader/service/paper/PaperTradeServiceTest.java

```diff
@@ -109,7 +111,7 @@ public void testPlaceOrderFundExceededMargin() {
             .orderStatus(Order.OrderStatus.NEW)
             .build();
 
-        paperTradeService.placeLimitOrder(order);
+        assertThrows(FundsExceededException.class, () -> paperTradeService.placeLimitOrder(order));
```

Comment:
This test throws an exception because we did not set the order leverage. 
We should have a test the happy scenario (with `order.getLeverage() != null` ) that does not throw an exception

Reply:
Thanks, you were right. We were missing copying the leverage value and I also created an unit test to assert on that